importer/import.go

```diff
@@ -10,6 +10,13 @@ import (
 	"github.com/ipfs/go-ipfs-cmdkit/files"
 )
 
+func shouldIgnore(err error) bool {
+	if strings.Contains(err.Error(), "dagservice: block not found") {
```

Comment:
I am unable to find where this error string is generated. I would like that our comparison is something like `if err == module.ErrNotFound`. But I've been looking around for a while and I don't find the place where this error is constructed like this. Can you trace this? I only find references to `blockstore: block not found`.

Reply:
Wow this is hard to find.  I will let you know if I make any progress here.

Reply:
Omg it was me the whole time

Reply:
haha and I did not have the branch checked out so my greps wouldn't find it!